gcs-object/test/integration/barebones/inspec.yml

```diff
@@ -0,0 +1,13 @@
+---
+name: default
+inspec_version: '>= 2.2.35'
+depends:
+  - name: inspec-gcp
+    url: https://github.com/inspec/inspec-gcp/archive/master.tar.gz
```

Comment:
I have not see that before but it is mentioned in the [InSpec Profile documentation](https://www.inspec.io/docs/reference/profiles/#inspec-yml). This should not be necessary when using Kitchen-Terraform as it handles the pinning of InSpec, but this would be necessary if running the profile directly with InSpec.

Reply:
This was pulled from the original source. I can remove it if we think that's better.